src/gatsby/src/components/park/campingDetails.js

```diff
@@ -73,8 +73,9 @@ export const CampingType = ({ camping, parkOperation }) => {
             {camping?.campingType?.campingTypeName}
           </h3>
         </div>
-        <HtmlContent ariaHidden={!expanded} className="park-camping-description">
-          {expanded ? campingDescription : collapsedDescription}
+        {/* let voiceover skip reading content if it is not expanded */}
+        <HtmlContent ariaHidden={hasExpandCondition ? !expanded : false} className="park-camping-description">
```

Comment:
Would this make sense to simplify these ternaries?

```suggestion
        <HtmlContent ariaHidden={hasExpandCondition && !expanded} className="park-camping-description">
```

Reply:
Great, yes that makes sense. Thanks!